test/user.test.js

```diff
@@ -1606,6 +1606,24 @@ describe('User', function() {
           done();
         });
 
+        it('returns same custom verify options after verify user model', function() {
+          const verifyOptons = {
+            type: 'email',
+            from: 'test@example.com',
+          };
+          User.settings.verifyOptions = Object.assign({}, verifyOptons);
+          const user = new User({
+            email: 'example@example.com',
+            password: 'pass',
+            verificationToken: 'example-token',
+          });
+          return user
+            .verify(User.getVerifyOptions())
```

Comment:
In one test, we should call `user.verify(verifyOptions)` - don't use `getVerifyOptions`.

In the second test, we should call `getVerifyOptions` and modify the returned value directly from the test, without calling `user.verify`.